src/lib.rs

```diff
@@ -1606,12 +1620,15 @@ pub mod pallet {
 		///
 		/// - `dispute_id`: The identifier of the dispute.
 		#[pallet::call_index(18)]
-		#[pallet::weight(323353000)]
-		pub fn report_vote_executed(origin: OriginFor<T>, dispute_id: DisputeId) -> DispatchResult {
+		#[pallet::weight(<T as Config>::WeightInfo::report_vote_executed(u8::MAX.into()))]
```

Comment:
The weight function parameter doesnt seem correct here, as the vote execution is only ever for a single round (dispute) at a time.

Reply:
This extrinsic is iterating over all the vote_rounds that's why we have this additional param for getting max vote rounds.